inflate.c

```diff
@@ -1504,7 +1504,7 @@ z_streamp strm;
 {
     struct inflate_state FAR *state;
 
-    if (strm == Z_NULL || strm->state == Z_NULL) return -1L << 16;
+    if (strm == Z_NULL || strm->state == Z_NULL) return (unsigned)-1L << 16;
```

Comment:
This patch was not applied, so no worries.  This issue was fixed differently in a recent commit on the develop branch.